manifests/custom_config.pp

```diff
@@ -23,8 +23,15 @@
 
   validate_bool($verify_config)
 
+  if $priority {
+    $priority_prefix = "${priority}-"
+  } else {
+    $priority_prefix = ''
+  }
```

Comment:
i would, as always, prefer this to be a [selector](https://docs.puppetlabs.com/puppet/3.7/reference/lang_conditional.html#selectors)